libs/designsystem/src/lib/components/radio/radio-group/radio-group.component.spec.ts

```diff
@@ -697,4 +709,510 @@ describe('RadioGroupComponent', () => {
       });
     });
   });
+
+  fdescribe('when used in a form', () => {
```

Comment:
Consider adding a describe block that tests the ControlValueAccessor interface in itself as we have in dropdown.component: https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts#L825

Reply:
Good point. I actually had this on my to do. See f24c2ca.